key,default_value
userAgent,"aws-cli/1.16.47 Python/2.7.13 Linux/4.17.0-3rodete2-amd64 botocore/1.12.37"
eventID,"4a51237d-9a32-4f71-9916-09b4f9adb6fc"
errorMessage,"User: arn:aws:iam::811596193553:user/backup is not authorized to perform: organizations:ListRoots on resource: *"
userIdentity_type,"IAMUser"
userIdentity_principalId,"AIDA9BO36HFBHKGJAO9C1"
userIdentity_arn,"arn:aws:iam::811596193553:user/backup"
userIdentity_accountId,"811596193553"
userIdentity_accessKeyId,"AKIA01U43UX3RBRDXF4Q"
userIdentity_userName,"backup"
eventType,"AwsApiCall"
errorCode,"AccessDenied"
sourceIPAddress,"245.70.252.112"
eventName,"ListRoots"
eventSource,"organizations.amazonaws.com"
recipientAccountId,"811596193553"
requestParameters,"None"
awsRegion,"us-east-1"
requestID,"f392a56a-e0fa-11e8-b3c3-af159c59891c"
responseElements,"None"
eventVersion,"1.04"
eventTime,"2018-11-05T13:01:46Z"
